_posts/2023-02-23-nfl-draft-value-chart/nfl-draft-value-chart.Rmd

```diff
@@ -0,0 +1,443 @@
+---
+title: "NFL Draft Value Chart"
+description: |
+  A short description of the post.
```

Comment:
I think this will serve as subtitle? I can't remember to be honest

Reply:
Oops